plugin.audio.mixcloud/resources/language/French/strings.xml

```diff
@@ -3,44 +3,44 @@
 	<!-- common -->
```

Comment:
Something you might want to consider for the future is to adapt the strings.xml to the new strings.po language layout: http://kodi.wiki/view/Language_support